test/lib/convert.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+require('../helpers');
+const lab = exports.lab = Lab.script();
+const convert = require('../../lib/convert');
+
+
+describe('convert', () => {
+  describe('ounces and teaspoons', () => {
+    it('should convert ounces to teaspoons', () => {
+      const results = convert(4, 'ounces', 'teaspoons');
+      expect(results).to.equal(24);
+    });
+    it('should convert teaspoons to ounces', () => {
+      const results = convert(24, 'teaspoons', 'ounces');
+      expect(results).to.equal('4.00');
+    });
+    it('should convert regardless of mix casing', () => {
+      const results = convert(4, 'Ounces', 'TeaspoonS');
+      expect(results).to.equal(24);
+    });
+  });
+  describe('cups and ml', () => {
+    it('should convert cups to ml', () => {
+      const results = convert(3, 'cups', 'ml');
+      expect(results).to.equal(750);
+    });
+    it('should convert cups to ml', () => {
+      const results = convert(750, 'ml', 'cups');
+      expect(results).to.equal(3);
+    });
+    it('should convert regardless of mix casing', () => {
+      const results = convert(750, 'mL', 'cuPs');
+      expect(results).to.equal(3);
+    });
+  });
+  describe('farenheit and celsius', ()  => {
+    it('should convert farenheit to celsius', () => {
+      const results = convert(375, 'farenheit', 'celsius');
+      expect(results).to.equal(191);
+    });
+    it('should convert celsius to farenheit', () => {
+      const results = convert(20, 'celsius', 'farenheit');
+      expect(results).to.equal(68);
+    });
+    it('should convert regardless of mix casing', () => {
+      const results = convert(20, 'Celsius', 'Farenheit');
+      expect(results).to.equal(68);
+    });
+  });
+  describe('return null if input is anything else', () => {
```

Comment:
This describe is a bit confusing. The idea of the test descriptions is that, when put together, it makes sense (the describe and the it). For example, in the describes and it above, you can read it out loud like this:

`farenheit and celcius should convert farenheit to celcius` (thats one test)
`farenheit and celcius should convert celcius to farenheit` (thats another test)

This one reads

`return null if input is anything else should return null`

A more descriptive describe could be `when there's no conversion available for the given input`, and then the it reads correctly `should return null`.  Put it together

`when there's no conversion available for the given input, it should return null`

More descriptive tests helps other devs picking up your code better understand the full scope of what is being achieved with a given code